express/src/middleware/sessionValidator.ts

```diff
@@ -0,0 +1,10 @@
+import {NextFunction, Request, Response} from "express";
+
+export default function sessionValidator(req: Request, res: Response, next: NextFunction) {
+    if (req.session.user || req.path === "/" || req.path === "/sign-in" || req.path === "/create/get-email") {
```

Comment:
We can't keep adding URLs here, this is not a sustainable approach. Instead only add this check to pages that require a session.

Reply:
Yes. I have removed this file in the next commit. So probably this does not need to be addressed.